run.py

```diff
@@ -312,7 +315,8 @@ def run(args):
             osp_cred = yaml.safe_load(osp_cred_stream)
 
     if cleanup_name is not None:
-        cleanup_ceph_nodes(osp_cred, cleanup_name)
+        instance_name = f"-{cleanup_name}-"
```

Comment:
IMO, if the user is providing this value with cleanup, we must allow support for partial matches as this is done with the user knowledge. I would like to not reduce this scope/behavior

Reply:
Done. Moved back to existing behavior